.github/workflows/consistency-functional.yml

```diff
@@ -126,7 +128,7 @@ jobs:
           RABBIT_USERID=guest
           RABBIT_PASSWORD=guest
           # Pre-requisite
-          ENABLED_SERVICES=mysql,key
```

Comment:
This worked before as it was running in the same container, in this case is external and we can restart it.